utils/utils.go

```diff
@@ -75,22 +75,19 @@ func CleanUpNamespace(args *skel.CmdArgs, logger *log.Entry) error {
 // CleanUpIPAM calls IPAM plugin to release the IP address.
 // It also contains IPAM plugin specific changes needed before calling the plugin.
 func CleanUpIPAM(conf NetConf, args *skel.CmdArgs, logger *log.Entry) error {
-	var err error
-
-	// Always try to release the address. Don't deal with any errors till the endpoints are cleaned up.
 	fmt.Fprintf(os.Stderr, "Calico CNI releasing IP address\n")
 	logger.WithFields(log.Fields{"paths": os.Getenv("CNI_PATH"),
 		"type": conf.IPAM.Type}).Debug("Looking for IPAM plugin in paths")
 
 	// We need to replace "usePodCidr" with a valid, but dummy podCidr string with "host-local" IPAM.
 	if conf.IPAM.Type == "host-local" && strings.EqualFold(conf.IPAM.Subnet, "usePodCidr") {
-
 		// host-local IPAM releases the IP by ContainerID, so podCidr isn't really used to release the IP.
 		// It just needs a valid CIDR, but it doesn't have to be the CIDR associated with the host.
 		dummyPodCidr := "0.0.0.0/0"
 		var stdinData map[string]interface{}
 
-		if err := json.Unmarshal(args.StdinData, &stdinData); err != nil {
+		err := json.Unmarshal(args.StdinData, &stdinData)
```

Comment:
this is because of line 97 where args.StdinData: `non-name args.StdinData on left side of :=
` so have to reuse the `err` variable.